kitsune/search/tasks.py

```diff
@@ -178,7 +178,7 @@ def unindex_task(cls, id_list, **kw):
 @task()
 @timeit
 def update_synonyms_task():
-    es = get_es()
+    es = get_es()  # noqa
```

Comment:
Is this being ignored by flake8 because it's not imported? I don't think we should paper over it like this as we might forget to remove it later.

Reply:
Yeah. I think I can add a comment here with `TODO:` to remove it.
I dont know any other easy way to make the flake8 happy for this.